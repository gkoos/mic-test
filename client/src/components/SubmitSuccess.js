import React from 'react';

const SubmitSuccess = props => {
    return (
        <h2>
            Thank you, we will contact you soon
        </h2>
    )
}

export default SubmitSuccess;