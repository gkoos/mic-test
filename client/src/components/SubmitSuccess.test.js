import React from 'react';
import SubmitSuccess from './SubmitSuccess';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
        .create(<SubmitSuccess/>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});