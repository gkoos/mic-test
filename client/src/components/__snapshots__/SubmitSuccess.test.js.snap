// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h2>
  Thank you, we will contact you soon
</h2>
`;
